client/src/components/pages/TeacherProfile/index.js

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from 'react';
+import './index.css';
+import TeacherImage from '../../../assets/teacher.jpg';
+import fakeData from './fakeData';
+
+class TeacherProfile extends Component {
+  state = {
+    data: {},
+    activeContent: `about`,
+  };
+
+  ActiveContentEnum = Object.freeze({
```

Comment:
this makes it a field of the class, i.e. `TeacherProfile.ActiveContentEnum`. The ActiveContentEnum is meant more like a (fake) type; anyways, in the future, it's better to define constants and enums outside of the class (for Javascript, in other languages it's different.)